src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/AprilTagCameraIO.java

```diff
@@ -0,0 +1,21 @@
+package frc.trigon.robot.poseestimation.apriltagcamera;
```

Comment:
Shouldn't this be in the io package?

Reply:
Double check how we used to do IO layers in the old code